### PR TITLE
adding I64/U64 support to the auto-batching

### PR DESCRIPTION
**Pull request by myshevts**
NB: in the API 1.0 the I32 was automatically used (by the CNNetwork?), so i64 inputs it didn't fail...

### CODE REVIEW

File: src/plugins/auto_batch/auto_batch.cpp [hunk 1]
```diff
@@ -109,7 +109,17 @@ void AutoBatchInferRequest::ShareBlobsWithBatchRequest() {
...
+                res = create_shared_blob_on_top_of_batched_blob<InferenceEngine::Precision::U64>(
+                        _myBatchedRequestWrapper._inferRequestBatched->GetBlob(it.first),
+                        _batchId,
+                        _batchSize);
+        case InferenceEngine::Precision::I64:
+            res = create_shared_blob_on_top_of_batched_blob<InferenceEngine::Precision::I64>(
+                    _myBatchedRequestWrapper._inferRequestBatched->GetBlob(it.first),
+                    _batchId,
+                    _batchSize);
+            break;
```
Comment (ilya-lavrenov) [inline src/plugins/auto_batch/auto_batch.cpp:122]: what tests are fixed?

Comment (ilyachur) [reply src/plugins/auto_batch/auto_batch.cpp:122]: Should we extend these tests for auto plugin? https://github.com/openvinotoolkit/openvino/blob/master/src/tests/functional/plugin/shared/include/behavior/ov_infer_request/io_tensor.hpp#L47

Comment (myshevts) [reply src/plugins/auto_batch/auto_batch.cpp]: these tests were not instantiated even for the GPU- so I've added them, notice that u16/i16 is fixed by the https://github.com/openvinotoolkit/openvino/pull/10180/ so it is commented in the tests

File: src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp [hunk 2]
```diff
@@ -69,6 +69,20 @@ std::vector<ov::element::Type> prcs = {
     ov::element::u64,
 };
 
+std::vector<ov::element::Type> supported_input_prcs = {
+        ov::element::boolean,
+        ov::element::f16,
+        ov::element::f32,
+        ov::element::i8,
+         // ov::element::i16,
```
Comment (myshevts) [inline src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp:77]: #10180 adds i16/u16 (along with a fix on the G{U plugin)

Comment (ilyachur) [reply src/tests/functional/plugin/gpu/shared_tests_instances/behavior/ov_infer_request/io_tensor.cpp:77]: Maybe will be better to use skip config to disab;e these tests

**Review by ilyachur — APPROVED** [review]

**Pull request merged**